test/components/app.spec.js

```diff
@@ -64,11 +67,13 @@ describe('App', () => {
             clock.tick(60000);
           })
           .respondWithData(() => '(v2024.1.3-sha)')
-          .afterResponse(app => {
-            clock.tick(0);
+          .afterResponse(async (app) => {
+            await clock.tickAsync(0);
```

Comment:
`tick()` needs to become `tickAsync()` in order to give the DOM a chance to update. We need `.alert-cta` to be rendered before we click it.